arielpartners/capdash2-server
# moved unit tests to test/unit structure, fixed all rubocop violations

TO TEST:
bundle update
rake test
see clean rubocop report
see clean test report
view reports/unit/results/index.html in a browser: should be clean
